src/Workspaces/Core/Portable/CodeFixes/CodeFixProvider.cs

```diff
@@ -19,6 +19,9 @@ public abstract class CodeFixProvider
         /// <summary>
         /// Computes one or more fixes for the specified <see cref="CodeFixContext"/>.
         /// </summary>
+        /// <remarks>
+        /// The behavior of this method is undefined if the the given <paramref name="context"/> contains <see cref="CodeFixContext.Diagnostics"/> which are not supported by this fixer, as indicated by <see cref="FixableDiagnosticIds"/> property.
```

Comment:
I suggest rewriting this in terms of the constraint that is required to hold, and not discussing what happens if the constraint does not hold.